ext/AvizExt/viz/scenario.jl

```diff
@@ -4,6 +4,21 @@
 
 Plot scenario outcomes over time.
```

Comment:
@ConnectedSystems 

Since you are editing this docstring: if I understand correctly, this function plots one specific metric for all the scenarios over time making a distinction between guided, unguided and no intervention. If that's correct, I think we could (at least) improve the description here a little bit..? (But I would argue that the name of this function could be changed to something like `scenarios_metric` or just `metric` maybe)

Reply:
I'm open to a rename but I disagree with the name change (at least the one suggested here).

`ADRIA.viz.scenario()` indicates you'd like to visualize the scenario (specifically, the `viz.scenario`).

How about `ADRIA.viz.scenario_outcomes()` ?

As for the description, I'm not sure what other text should be included. What were you thinking of adding?

Reply:
My comment was more directed to the docstring than to the name of the function actually. I am more asking a question and only suggesting a change depending on the answer. What is the difference between a metric and an outcome? My confusion was that, because the variable `y` represent the "results of scenario metric", and we are plotting `y`, then to me we are plotting the scenario metric. 

But I don't think this is a blocker so if you prefer, I can approve this PR and we can have this discussion separately.